cmake/Platform/Linux/LYWrappers_linux.cmake

```diff
@@ -39,3 +48,27 @@ function(ly_handle_custom_output_directory target output_subdirectory)
     endif()
 
 endfunction()
+
+#! ly_apply_debug_strip_options: Apply debug stripping options to the target output for non-debug configurations.
+#
+#\arg:target Name of the target to perform a post-build stripping of any debug symbol)
+function(ly_apply_debug_strip_options target)
+
+    find_program(LLVM_STRIP_TOOL llvm-strip)
+    if (NOT LLVM_STRIP_TOOL)
+        return()
+    endif()
+
+    if (NOT ${LY_STRIP_DEBUG_SYMBOLS})
+        return()
+    endif()
+
+    add_custom_command(TARGET ${target} POST_BUILD
+        COMMAND "${CMAKE_COMMAND}" -P "${LY_ROOT_FOLDER}/cmake/Platform/Linux/StripDebugSymbols.cmake"
```

Comment:
`StripDebugSymbols.cmake` isn't part of this change? I don't see it in current development branch either.

Reply:
Forgot to add that important piece